packages/vue/src/components/sign-in.vue

```diff
@@ -31,41 +33,31 @@ const { getForgotPasswordText, getSignInText, getSigningInText } =
 // Computed Properties
 const forgotYourPasswordLink = computed(() => getForgotPasswordText());
 const signInButtonText = computed(() => getSignInText());
-const signIngButtonText = computed(() => getSigningInText());
-
-const { state, send } = useAuth();
-const actorState = computed(() =>
-  getActorState(state.value)
-) as ComputedRef<SignInState>;
+const signingInButtonText = computed(() => getSigningInText());
 
 // Methods
-
 const onInput = (e: Event): void => {
   const { name, value } = e.target as HTMLInputElement;
-  send({
-    type: 'CHANGE',
-    //@ts-ignore
-    data: { name, value },
-  });
+  updateForm({ name, value });
 };
 
 const onSignInSubmit = (e: Event): void => {
+  // TODO(BREAKING): remove unused emit
+  // istanbul ignore next
```

Comment:
I am deliberately ignoring coverage for 

```ts
if (attrs?.onSignInSubmit)
```

because it's an unofficial, undocumented API that we do not test or use.

Reply:
mstm. Should we mark it as `@deprecated`?

Reply:
Sounds good, done in [4d10a42](https://github.com/aws-amplify/amplify-ui/pull/4075/commits/4d10a4249896c35cc4da43e28ead38805ff8787a)!